contracts/RMRK/extension/tokenProperties/RMRKPropertiesAccessControl.sol

```diff
@@ -0,0 +1,251 @@
+
+        emit AccessControlUpdated(
+            collection,
+            parameterType,
+            parameterId,
+            accessType,
+            collaboratorAddresses,
+            collaboratorAddressAccess,
+            specificAddresses,
+            specificAddressAccess
+        );
+    }
+
+    /**
+     * @inheritdoc IRMRKPropertiesAccessControl
+     */
+    function isCollaborator(
+        address collaborator,
+        address collection,
+        ParameterType parameterType,
+        uint256 parameterId
+    ) external view returns (bool) {
+        return
+            _accessControl[collection][parameterType][parameterId]
+                .collaboratorAddresses[collaborator];
+    }
+
+    /**
+     * @inheritdoc IRMRKPropertiesAccessControl
+     */
+    function isSpecificAddress(
+        address specificAddress,
+        address collection,
+        ParameterType parameterType,
+        uint256 parameterId
+    ) external view returns (bool) {
+        return
+            _accessControl[collection][parameterType][parameterId]
+                .specificAddresses[specificAddress];
+    }
+
+    /**
+     * @notice Modifier to check if the caller is authorized to call the function.
+     * @dev If the authorization is set to TokenOwner and the tokenId provided is of the non-existent token, the
+     *  execution will revert with `ERC721InvalidTokenId` rather than `RMRKNotTokenOwner`.
+     * @dev The tokenId parameter is only needed for the TokenOwner authorization type, other authorization types ignore
+     *  it.
+     * @param collection The address of the collection.
+     * @param parameterType The type of the parameter.
+     * @param parameterId The ID of the parameter.
+     * @param tokenId The ID of the token.
+     */
+    modifier onlyAuthorizedCaller(
+        address collection,
+        ParameterType parameterType,
+        uint256 parameterId,
+        uint256 tokenId
+    ) {
+        _onlyAuthorizedCaller(collection, parameterType, parameterId, tokenId);
+        _;
+    }
+
+    /**
+     * @notice Modifier to check if the collection is registered.
+     * @param collection Address of the collection.
+     */
+    modifier onlyRegisteredCollection(address collection) {
+        if (!_issuerSettings[collection].registered) {
+            revert RMRKCollectionNotRegistered();
+        }
+        _;
+    }
+
+    /**
+     * @notice Modifier to check if the collection is not registered.
+     * @param collection Address of the collection.
+     */
+    modifier onlyUnregisteredCollection(address collection) {
+        if (_issuerSettings[collection].registered) {
+            revert RMRKCollectionAlreadyRegistered();
+        }
+        _;
+    }
+
+    /**
+     * @notice Modifier to check if the caller is the issuer of the collection.
+     * @param collection Address of the collection.
+     */
+    modifier onlyIssuer(address collection) {
+        if (_issuerSettings[collection].useOwnable) {
```

Comment:
I think it's a laudable effort but to dangerous to keep. Let's only support ownable IMO

Reply:
Why do you think this is dangerous?

Reply:
Allowing anyone to claim a way of ownership can lead to ransoms or phishing attempts. You could more easily convince holders that you are the rightful issuer by modifying the attributes of  a well known centralized attributes repo.

Reply:
While this diminishes the list of potential collection users, I agree with you on the risks. It simplifies things from our end and makes the smart contract more reliable 🤔